dotnet/src/Connectors/Connectors.OpenAI.UnitTests/Services/OpenAIChatCompletionServiceTests.cs

```diff
@@ -98,13 +95,10 @@ public async Task ItUsesCustomEndpointsWhenProvidedDirectlyAsync(string endpoint
     }
 
     [Theory]
-    [InlineData("http://localhost:1234/chat/completions", "http://localhost:1234/chat/completions")] // Uses full path when provided
-    [InlineData("http://localhost:1234/v2/chat/completions", "http://localhost:1234/v2/chat/completions")] // Uses full path when provided
-    [InlineData("http://localhost:1234", "http://localhost:1234/v1/chat/completions")]
+    [InlineData("http://localhost:1234/v1/chat/completions", "http://localhost:1234/v1/chat/completions")] // Uses full path when provided
+    [InlineData("http://localhost:1234/", "http://localhost:1234/v1/chat/completions")]
     [InlineData("http://localhost:8080", "http://localhost:8080/v1/chat/completions")]
     [InlineData("https://something:8080", "https://something:8080/v1/chat/completions")] // Accepts TLS Secured endpoints
-    [InlineData("http://localhost:1234/v2", "http://localhost:1234/v2/chat/completions")]
-    [InlineData("http://localhost:8080/v2", "http://localhost:8080/v2/chat/completions")]
```

Comment:
Why are we removing some test cases?

Reply:
They dont work anymore with the latest changes from OpenAI SDK.